xrootd.sh

```diff
@@ -23,6 +23,14 @@ XROOTD_PYTHON=""
 PYTHON_EXECUTABLE=$(/usr/bin/env python3 -c 'import sys; print(sys.executable)')
 PYTHON_VER=$( ${PYTHON_EXECUTABLE} -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")' )
 
+# Report versions of pip and setuptools
+echo "###################
+pip version:
+$(python3 -m pip -V)
+setuptools version:
+$(python3 -m pip show setuptools | grep 'Version\|Location')
+###################"
+
```

Comment:
This looks like temporary debugging info -- can you remove this from the final recipe?

Reply:
yes, but wouldn't you like to have this debugging output permanent? it would tremendously help with reports from users where system or user side version could interfere with alibuild ones ..

Reply:
Sure, fine by me if you find it useful!